packages/ensnode-schema/src/resolver-records.schema.ts

```diff
@@ -1,66 +1,82 @@
  *
- * This file specifies the following datamodel:
- *
- * Resolver <- subgraph schema entity, actually (Resolver contract, Node)
- * - name <- added, implicitly keyed by Node
- * - abi, contenthash, pubkey, version, ...etc if desired
- * - has many ResolverAddressRecords by `coinType`
- *   - coinType
- *   - address
- * - has many ResolverTextRecords by `key`
- *   - key
- *   - value
+ * Naturally, this discrepancy may be confusing, but throughout this codebase a Resolver _entity_
+ * refers to that _relationship_ between a Resolver and Node, _not_ the unique Resolver contract itself.
  *
- * This results in the following query patterns (ponder graphql pseudo code) to look up the records
- * for a Node within a Resolver contract:
+ * The Subgraph describes the following datamodel instead:
+ * Resolver (the Resolver-Node relationship)
+ *   - has one `addr` record for a Node
+ *   - has one `contenthash` record for a Node
+ *   - has one set of `coinTypes` storing the keys of all ENSIP-9 address records emitted for a Node
+ *   - has one set of `texts` storing the keys of all ENSIP-5 text records emitted for a Node
  *
- * NOTE: generate the (subgraph entity) Resolver ID as:
- * - (if running in subgraph-compat mode): `${resolverContractAddress.toLowerCase()}-${node}`
- * - (if not running in subgraph-compat mode): `${chainId}-${resolverContractAddress.toLowerCase()}-${node}`
+ * In this file, we extend the subgraph's Resolver _entity_ with the following:
+ * Resolver (the Resolver-Node relationship)
+ *   + has one `name` record for a Node (see ENSIP-3)
+ *   + has many `addressRecords` for a Node (by coinType) (see ENSIP-9)
+ *     - ResolverAddressRecord indexes `coinType` and `address`
+ *   + has many `textRecords` for a Node (by key) (see ENSIP-5)
+ *     - ResolverTextRecord indexes `key` and `value`
  *
+ * This results in the following query patterns for the autogenerated GraphQL APIs:
  * @example
  * ```gql
- * query RecordsForNode($id: ID!) {
+ * # NOTE: generate the Resolver entity ID as:
+ * # - (if running in subgraph-compat mode): `${resolverContractAddress.toLowerCase()}-${node}`
+ * # - (if not running in subgraph-compat mode): `${chainId}-${resolverContractAddress.toLowerCase()}-${node}`
+ * query RecordsForNodeOnResolver($id: ID!) {
  *   resolver(id: $id) {
+ *     addr
  *     name
- *     addressRecords { totalCount items { coinType address } }
- *     textRecords { totalCount items { key value } }
+ *     addressRecords {
+ *       totalCount
+ *       items {
+ *         coinType     # see ENSIP-9, ENSIP-11
+ *         address      # see ENSIP-9, ENSIP-11
+ *       }
+ *     }
+ *     textRecords {
+ *       totalCount
+ *       items {
+ *         key          # see ENSIP-5, ENSIP-12, ENSIP-18
+ *         value        # see ENSIP-5, ENSIP-12, ENSIP-18
+ *       }
+ *     }
  *   }
  * }
  *```
+ *
+ * NOTE: The addition of these record keys and values does NOT allow the caller to confidently
```

Comment:
```suggestion
 * NOTE: These "additional resolver records" do NOT allow the caller to confidently
```

Suggesting this to help maximize explicit relationships between terminology we introduce.